testng.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="Web Automation Tests" preserve-order="true" configfailurepolicy="continue" parallel="methods" thread-count="1">
```

Comment:
thread-count="5"

Reply:
Facing issue with parallel execution. Will be addressed in upcoming PR. Added to **Upcoming changes** section of the PR description as well